### PR TITLE
08-Calling-APIs

### PR DESCRIPTION
**Pull request by volbap**
- Added `08-calling-apis.md` quickstart.
- Updated `index.yml`.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/ios-swift/08-calling-apis.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Calling APIs
+description: This tutorial will show you how to manage tokens to make authenticated API calls, using NSURLSession.
+---
+
+The reason for implementing authentication, in the first place, is to protect information. In this case, your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. We recommend that you conform to RFC standards by sending valid authentication tokens through an authorization header.
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/08-calling-apis.md:16]: @pablolvillar The doc is missing the system requirements and sample download link.

File: articles/native-platforms/ios-swift/08-calling-apis.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+let url = NSURL(string: "your api url")!
+let request = NSMutableURLRequest(URL: url)
+// Configure your request here (method, body, etc)
+```
+
+Once you send a request and your API returns a response, its status code is going to be displayed in an alert view.
+
+> You can use your own API to test, or, if you want, you can use [this sample server](https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample/tree/master/Server), which is quickly configurable.
+>
+> For further information on authentication API on the server-side, check [the official documentation](https://auth0.com/docs/api/authentication).
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/08-calling-apis.md]: @pablolvillar Please make this link relative: `/api/authentication`

**Comment by tenmaster** [comment]
+1 @ntotten @mpaktiti @chenkie

**Comment by mpaktiti** [comment]
@pablolvillar Can you please review my comments and apply the fixes? Thanks.

**Comment by volbap** [comment]
@mpaktiti Done with the fixes.

**Comment by mpaktiti** [comment]
Before deployment:
- Seed project should be merged https://github.com/auth0-samples/auth0-ios-swift-sample/pull/11
- Sample is at auth0-samples repo, download zip functionality should be in place.
- `index.yml` should be checked again

**Comment by volbap** [comment]
Closing this pull request. These changes will be included in a new pull request.

**Pull request closed without merging**